package.json

```diff
@@ -5,26 +5,37 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "ncc build src --license licenses.txt",
-    "lint": "eslint ."
+    "test": "jest",
+    "lint": "eslint .",
+    "prepare": "husky install"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/estherixz/github-action-notify-release.git"
+    "url": "git+https://github.com/nearform/github-action-notify-release.git"
   },
   "author": "Esther Saimpou",
+  "contributors": [
+    "Sameer Srivastava <sameer.srivastava@nearform.com>"
+  ],
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/estherixz/github-action-notify-release/issues"
+    "url": "https://github.com/nearform/github-action-notify-release/issues"
   },
-  "homepage": "https://github.com/estherixz/github-action-notify-release#readme",
+  "homepage": "https://github.com/nearform/github-action-notify-release#readme",
   "dependencies": {
     "@actions/core": "^1.2.7",
-    "@actions/exec": "^1.0.4",
     "@actions/github": "^4.0.0",
     "date-fns": "^2.21.1"
   },
   "devDependencies": {
     "@vercel/ncc": "^0.28.3",
-    "eslint": "^7.24.0"
+    "eslint": "^7.25.0",
+    "husky": "^6.0.0",
+    "jest": "^26.6.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm test && npm run lint && npm run build && git add dist"
```

Comment:
remove test and lint, we don't need them as they unnecessarily slow down the commits. we only want to build

Reply:
Done